core/src/components/item/item.md.scss

```diff
@@ -292,3 +321,97 @@
 :host(.item-label-color) {
   --highlight-color-focused: #{current-color(base)};
 }
+
+:host(.item-fill-solid.ion-color),
+:host(.item-fill-outline.ion-color) {
+  --highlight-background: #{current-color(base)};
+  --highlight-color-focused: #{current-color(base)};
+}
+
+// Material Design Item: Fill Solid
+// --------------------------------------------------
+
+:host(.item-fill-solid) {
+  --background: #{$item-md-input-fill-solid-background-color};
+  --background-hover: #{$item-md-input-fill-solid-background-color-hover};
+  --background-focused: #{$item-md-input-fill-solid-background-color-focus};
+  --border-width: 0 0 #{$item-md-border-bottom-width} 0;
+  --inner-border-width: 0;
+
+  @include border-radius(4px, 4px, 0, 0);
+}
+
+:host(.item-fill-solid) .item-native {
+  --border-color: #8f8f8f;
+}
+
+:host(.item-fill-solid) .item-native:hover {
+  --background: var(--background-hover);
+  --border-color: var(--highlight-color-hover);
+}
+
+:host(.item-fill-solid.item-has-focus) .item-native {
+  --background: var(--background-focused);
+  border-bottom-color: var(--highlight-color-focused);
+}
+
+:host(.item-fill-solid.item-shape-round) {
+  @include border-radius(16px, 16px, 0, 0);
+}
+
+// Material Design Item: Fill Outline
+// --------------------------------------------------
+
+:host(.item-fill-outline) {
+  --border-color: #9e9e9e;
```

Comment:
We should use a step color here so it looks good in dark mode:

```
var(--ion-color-step-500, #9e9e9e);
```

 not sure if 500 is the correct one, but you'll have to play around with it.